src/draw/DefaultDrawer.js

```diff
@@ -494,6 +497,46 @@ class DefaultDrawer {
     node.select('.component-container>rect').attr('data-parentId', ({ data }) => data.id);
   }
 
+  /**
+   * Pre-compute a node's height and width
+   *
+   * @param {Node} item - The node
```

Comment:
can you specify more, like D3 node something else

Reply:
Not really, since d3 seems to use the native Node object ( [https://developer.mozilla.org/en-US/docs/Web/API/Node](https://developer.mozilla.org/en-US/docs/Web/API/Node)) when building the hierarchy.

Reply:
Think you can improve it like the other docs: `@param {Node} component - The component to get the width for.`